app/models/foreman_docker/docker.rb

```diff
@@ -131,17 +132,19 @@ def bootstrap(args)
     end
 
     def client
-      @client ||= ::Fog::Compute.new(
+      cli = ::Fog::Compute.new(
```

Comment:
Because [we need to call `new` to set `Docker.url`](https://github.com/fog/fog/blob/master/lib/fog/fogdocker/compute.rb#L43).

I think the real solution would be to fix fog so that on each request it sets `Docker.url` to the client's docker_url.